terracumber_config/tf_files/Uyuni-Master-K3S.tf

```
@@ -137,7 +137,7 @@ module "cucumber_testsuite" {
         memory = 16384
       }
       runtime = "k3s"
-      container_repository = "registry.opensuse.org/systemsmanagement/uyuni/master/containers/uyuni"
+      container_repository = "registry.opensuse.org/systemsmanagement/uyuni/master/containers_leap_15.6/uyuni"
```

```suggestion
      container_repository = "registry.opensuse.org/systemsmanagement/uyuni/master/containers_leap_15.6"
```
If I am not mistaken. Same for the rest.

---

Ok, because mgradm is prepending the uyuni part too since last versions right?